hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/DelayedRequeueExceptionStrategy.java

```diff
@@ -59,11 +61,18 @@ protected boolean isUserCauseFatal(final Throwable cause) {
     }
 
     private boolean invalidMessage(final Throwable cause) {
-        return doesNotExist(cause) || cause instanceof TooManyStatusEntriesException
-                || cause instanceof InvalidTargetAddressException || cause instanceof ToManyAttributeEntriesException;
+        return doesNotExist(cause) || quotaHit(cause) || invalidContent(cause);
```

Comment:
After a small discussion with @michahirsch , we believe it's better to add the MessageHandlingException